scala/scala.bzl

```diff
@@ -61,7 +56,10 @@ _implicit_deps = {
         default = Label("@bazel_tools//tools/jdk:current_java_runtime"),
         cfg = "host"),
     "_java_runtime": attr.label(
-        default = Label("@bazel_tools//tools/jdk:current_java_runtime"))
+        default = Label("@bazel_tools//tools/jdk:current_java_runtime")),
+    "_scalac": attr.label(
```

Comment:
`_scalac` is a bit confusing, since it includes everything, what about `_scala_tools` or something?

Also, if we make this public, e.g. use `"scala_provider"` as the name with a default, users could change it at individual call sites, something that might be useful, not sure.

Reply:
Currently there's no point in making it public as we can only have one version of `@io_bazel_rules_scala//src/java/io/bazel/rulesscala/scalac:scalac_worker` (= only one version of scala) but for the future when we'd support multiple scala versions that's what I'd want to do.

I'll change the name to `_scala_provider` but keep it private for now